controllers/openstackcluster_controller.go

```diff
@@ -264,28 +264,30 @@ func reconcileNormal(ctx context.Context, scope *scope.Scope, patchHelper *patch
 		return reconcile.Result{}, err
 	}
 
-	availabilityZones, err := computeService.GetAvailabilityZones()
-	if err != nil {
-		return ctrl.Result{}, err
-	}
-
-	// Create a new list to remove any Availability
-	// Zones that have been removed from OpenStack
+	// Create a new list to remove any Availability Zones that have been removed from OpenStack
```

Comment:
don't fully understand this sentence...

Reply:
This comment was there before my change…

I think what it is trying to say is that we rebuild the AZ list instead of using a cached list in case the AZs have changed.